_includes/filezilla.md

```diff
@@ -0,0 +1,27 @@
+## General instructions
+
+The raw data of all studies published in IDR can be downloaded using the FTP. We recommend using the [FileZilla client](https://filezilla-project.org/download.php).
+
+You can download both whole studies as well as parts/single images.
+
+The ftp host is available using the following details:
+
+- host: `ftp.ebi.ac.uk/pub/databases/IDR/`
```

Comment:
I think in the standard FTP terminology the host is `ftp.ebi.ac.uk` while `/pub/databases/IDR` is the default remote directory. Using the Site Manager, this is how the configuration would be set-up
<img width="1168" alt="Screenshot 2023-12-01 at 11 32 27" src="https://github.com/IDR/idr.openmicroscopy.org/assets/1355463/f4f56bc3-2314-46a7-8518-bce883e5b9ab">
<img width="1168" alt="Screenshot 2023-12-01 at 11 32 31" src="https://github.com/IDR/idr.openmicroscopy.org/assets/1355463/47108f8d-ceaf-4dfa-aea3-dce8995be986">

Reply:
I don't think there is much value to be honest. Since it's anonymous connection and the quickconnect workflow nicely parses host + remote location, we should advertise this.

I consider storing a permanent reference to the FTP site on Filezilla outside the scope of this document.